Tema2/script.js

```diff
@@ -0,0 +1,73 @@
+"use strict";
+//motocicleta 200 cmc -> 8 lei
+//autobuz 200 cmc -> 26 lei
+//camion 200 cmc -> 32 lei
+// autoturism hibrid -> 50% reducere
+
+const localitate = prompt("Localitate");
```

Comment:
Pentru fiecare input (localitate, cilindre, hibrid, vehicul) as face la inceput, inainte sa intru in if-uri, mai multe if-uri in care verific daca datele sunt introduse corect. 
Cumva, nu execut un program pana nu ma asigur ca datele cu care o sa opereze programul sunt corecte. Astfel evitam acele else-uri, aproape asemanatoare, de mai jos cu **Datele introduse nu sunt corecte!**